### PR TITLE
🐛Source Zendesk Support: Add missing cursors

### LINKED ISSUE #19266: Zendesk Support Missing Cursors
## Expected Behavior
As a GA connector, Zendesk Support should have a cursor enabled for all possible streams.  From some cursory scoping it appears a few streams are missing cursors.  Below is that list for Zendesk Support (stream_name:cursor_name)

Streams with possible Cursors:
**custom_roles**: updated_at
**schedules**: updated_at
**sla_policies**: updated_at
**tickets**: updated_at
**users**: updated_at

The request here is if possible to add cursors for the above streams. 
@YowanR FYI

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #19266

## Bug Fix
Added cursors to the supporting streams mentioned in issue #19266 

Streams with possible Cursors:
custom_roles: updated_at
schedules: updated_at
sla_policies: updated_at
tickets: updated_at
users: updated_at

Changed the abnormal state, and incremental catalog for addition

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed
+

### CODE REVIEW

**Comment by btkcodedev** [comment]
Update: Awaiting tests creds for testing changes

**Comment by emilmar** [comment]
Great job! @btkcodedev Would it be simple to add TicketMetric to this PR too, as reported in the comments in #19266, or is better in another PR?

**Comment by btkcodedev** [comment]
Update: I could not test the changes as its gettings timeout every time as my internet connection is bad I think :(
Reviewers, please forward with the review.

**Comment by jrolom** [comment]
@marcosmarxm surfacing this one for review

**Comment by sajarin** [comment]
Ran this locally and some of the unit tests were failing. Running the tests here above.

**Comment by marcosmarxm** [comment]
@btkcodedev please fix the unit tests you can run they locally without creds.

**Comment by marcosmarxm** [comment]
@btkcodedev please fix the conflict and we're going to return the review to the contribution.

**Comment by btkcodedev** [comment]
Done :)

**Comment by sajarin** [comment]
Running integration tests in the PR above

File: airbyte-integrations/connectors/source-zendesk-support/metadata.yaml [hunk 1]
```diff
@@ -7,7 +7,8 @@ data:
   connectorType: source
   maxSecondsBetweenMessages: 10800
   definitionId: 79c1aa37-dae3-42ae-b333-d1c105477715
-  dockerImageTag: 0.10.1
+  dockerImageTag: 0.4.1
+  dockerImageTag: 0.10.2
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-zendesk-support/metadata.yaml:11]: Fixing merge issue
```suggestion
  dockerImageTag: 0.10.2
```

**Comment by btkcodedev** [comment]
@sh4sh Merge conflicts cleared, Please continue further review

**Comment by btkcodedev** [comment]
Closed as solved in https://github.com/airbytehq/airbyte/pull/30134

**Pull request closed without merging**